src/main/java/no/hyper/imagecrop/ImageTray.java

```diff
@@ -0,0 +1,205 @@
+package no.hyper.imagecrop;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.Point;
+import android.graphics.Rect;
+import android.support.v4.view.GestureDetectorCompat;
+import android.util.AttributeSet;
+import android.util.Log;
+import android.view.GestureDetector;
+import android.view.MotionEvent;
+import android.view.ScaleGestureDetector;
+import android.view.View;
+
+/**
+ * Created by jean on 07/10/15.
+ */
+class ImageTray extends View {
+
+    private Context context;
+    private Bitmap picture;
+    private Point middle;
+    private Paint bitmapPaint;
+    private Rect cropRect;
+    private int screenWidth;
+    private int screenHeight;
+    private int cropSize = 500;
```

Comment:
Yes that would be better indeed. Also add verification to call `setCropSize` only if cropSize is different of the default value ?